torch/csrc/jit/ir/scope.cpp

```diff
@@ -123,7 +123,10 @@ std::vector<InlinedCallStackEntry> InlinedCallStack::vec() {
   std::vector<InlinedCallStackEntry> r;
   c10::optional<InlinedCallStackPtr> current = intrusive_from_this();
   while (current) {
-    r.emplace_back(std::make_pair((*current)->fn_, (*current)->source_range_));
+    r.emplace_back(std::make_tuple(
+        (*current)->fn_,
+        (*current)->source_range_,
```

Comment:
Since we already have source_range_ in the reaching range, it would be great to include this in the debug info of bytecode.